cryptid_anchor/client/packages/idl/check_pass.ts

```diff
@@ -0,0 +1,295 @@
+export type CheckPass = {
```

Comment:
Is this location the default that the anchor command generates to? I'm tempted to not commit this file, but always generate it with a pre-build command.